internal/util/paramtable/base_table.go

```diff
@@ -142,28 +143,35 @@ func (gp *BaseTable) initConfigsFromLocal(formatter func(key string) string) {
 }
 
 func (gp *BaseTable) initConfigsFromRemote(formatter func(key string) string) {
-	endpoints, err := gp.mgr.GetConfig("etcd.endpoints")
-	if err != nil {
-		log.Info("cannot find etcd.endpoints")
+	etcdConfig := EtcdConfig{}
+	etcdConfig.Init(gp)
+	if etcdConfig.Endpoints.GetValue() == "" {
 		return
 	}
-	rootPath, err := gp.mgr.GetConfig("etcd.rootPath")
-	if err != nil {
-		log.Info("cannot find etcd.rootPath")
+	if etcdConfig.UseEmbedEtcd.GetAsBool() && !etcd.HasServer() {
```

Comment:
I think this line cause all the other places get messy. Maybe the dependency of etcd shall be removed here